ambry-messageformat/src/main/java/com.github.ambry.messageformat/MessageFormatRecord.java

```diff
@@ -45,7 +45,7 @@
   public static final short UserMetadata_Version_V1 = 1;
   public static final short Blob_Version_V1 = 1;
   public static final short Blob_Version_V2 = 2;
-  public static final short Metadata_Content_Version_V1 = 1;
+  // Metadata_Content_Version_V1 was unused and was removed to simplify the range request handling code.
```

Comment:
I think saying "Metadata_Content_Version_V1 was unused and was removed" is sufficient.